core/output/telegram.js

```diff
@@ -0,0 +1,43 @@
+var options = {
+    // Used by the monkey patch to node-telegram-bot-api
+    stopPolling: true
+};
+var TelegramBot = require.safe('node-telegram-bot-api');
+// Adds the missing stopPolling() method
+// https://github.com/yagop/node-telegram-bot-api/pull/51#issuecomment-217395990
+require.safe('monkey-patches-node-telegram-bot-api')(TelegramBot, options);
+
+var bot = null,
+    shim = require('../shim.js'),
+    api = null;
```

Comment:
instead of multiple `var` blocks, could these be combined into something like:
```javascript
var options = {
        // Used by the monkey patch to node-telegram-bot-api
        stopPolling: true
    },
    TelegramBot = require.safe('node-telegram-bot-api'),
    bot = null,
    shim = require('../shim.js'),
    api = null;

// Adds the missing stopPolling() method
// https://github.com/yagop/node-telegram-bot-api/pull/51#issuecomment-217395990
require.safe('monkey-patches-node-telegram-bot-api')(TelegramBot, options);
```

It just seems neater.